dotcom-rendering/src/web/lib/cardWrappers.tsx

```diff
@@ -403,6 +403,76 @@ export const Card33Media33 = ({
 		/>
 	);
 };
+/**
+ * ┏━━━━━━━━━━━━┱┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┐
+ * ┃▒▒▒▒▒▒▒▒▒▒▒▒┃           66%          ┊
+ * ┃▒▒▒▒▒▒▒▒▒▒▒▒┃        Remaining       ┊
+ * ┃            ┃                        ┊
+ * ┗━━━━━━━━━━━━┹┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┘
+ * Card designed to take up 33% of the container, with media that takes up 33%
```

Comment:
Given that the two card wrappers are almost identical, it could be useful to highlight the point where they differ? (i.e. the image position on mobile)

Reply:
Hey Pete, have tried to do that with the naming of the const (`Card33Media33Sublinks` vs `Card33Media33MobileTopSublinks` and in the options section as well: 
`Small image on the top (left on mobile)` vs `- Small image at the top, including on mobile`

Does that help? Or is there another way we can highlight the difference?

Reply:
I definitely think those are useful! But it took me a minute to 'spot the difference' in the options section.

I guess I'd be tempted to make it really explicit for future devs, by adding something to the top of each comment like "Use `Card33Media33MobileTopSublinks` instead if you need the image to always be at the top." But whatever feels right to you 👍